pkgs/by-name/tr/triton-llvm/package.nix

```
@@ -125,8 +125,12 @@ stdenv.mkDerivation (finalAttrs: {
     ncurses
   ];
 
+  # https://github.com/llvm/llvm-project/issues/56336
   preConfigure = ''
     cd llvm
+    cmakeFlagsArray+=(
+      -DLLVM_LIT_ARGS="-svj$((NIX_BUILD_CORES < 4 ? $((NIX_BUILD_CORES > 0 ? NIX_BUILD_CORES : 4)) : 4)) --no-progress-bar"
```

This could be simplified, right?

```suggestion
      -DLLVM_LIT_ARGS="-svj$((NIX_BUILD_CORES > 0 && NIX_BUILD_CORES < 4 ? $NIX_BUILD_CORES : 4)) --no-progress-bar"
```

---

Yup, thanks!